### DISCUSSION TITLE
Features

### DISCUSSION THREAD

**Discussion by bipon68** [Bipon-Dev/kpi-alumni-frontend#22] [Announcements]
Features

**Comment by bipon68** [comment]
Role permission
![image](https://github.com/user-attachments/assets/85e56294-36e7-4c57-a33e-3ad0a1c2fd68)

**Comment by bipon68** [comment]
Login
![image](https://github.com/user-attachments/assets/59ade0ef-3024-4609-bf00-7e2b0ebc916c)

**Comment by bipon68** [comment]
Register user
![image](https://github.com/user-attachments/assets/f38e0108-5485-4699-8e37-06809ac20d25)

**Comment by bipon68** [comment]
Access denied for delete
![image](https://github.com/user-attachments/assets/9664a438-a84e-4b4b-8f59-a00bae223b3c)

**Comment by Anindoroyapu** [comment]
#Members Filter System

![image](https://github.com/user-attachments/assets/fbd25895-f70d-48b9-b947-d25cf799df81)
![image](https://github.com/user-attachments/assets/76720f7d-1249-40d4-858d-6766e6edbaf8)

**Comment by bipon68** [comment]
Email format
![image](https://github.com/user-attachments/assets/a80bffeb-72f0-4790-9ba6-c4ceb4767993)
